easier68k/core/opcodes/eor.py

```diff
@@ -0,0 +1,278 @@
+
+        # repeat for the dest
+        if self.dest.mode in [EAMode.AbsoluteLongAddress]:
+            to_increment += OpSize.LONG.value
+
+        if self.dest.mode in [EAMode.AbsoluteWordAddress]:
+            to_increment += OpSize.WORD.value
+
+        result_unsigned = src_val.get_value_unsigned() ^ dest_val.get_value_unsigned()
+
+        msb_bit = 0
+
+        if self.size is OpSize.BYTE:
+            msb_bit = 0x80
+        elif self.size is OpSize.WORD:
+            msb_bit = 0x8000
+        elif self.size is OpSize.LONG:
+            msb_bit = 0x80000000
+
+        simulator.set_condition_status_code(ConditionStatusCode.N, msb_bit & result_unsigned != 0)
+        simulator.set_condition_status_code(ConditionStatusCode.Z, result_unsigned == 0)
+        simulator.set_condition_status_code(ConditionStatusCode.V, False)
+        simulator.set_condition_status_code(ConditionStatusCode.C, False)
+
+        # and set the value
+        self.dest.set_value(simulator, MemoryValue(OpSize.LONG, unsigned_int=result_unsigned))
+
+        # set the program counter value
+        simulator.increment_program_counter(to_increment)
+
+    def __str__(self):
+        return 'Eor command: size {}, src {}, dest {}'.format(self.size, self.src, self.dest)
+
+    @classmethod
+    def command_matches(cls, command: str) -> bool:
+        """
+        Checks whether a command string is an instance of this command type
+        :param command: The command string to check (e.g. 'MOVE.B', 'LEA', etc.)
+        :return: Whether the string is an instance of this command type
+        """
+        return opcode_util.command_matches(command, 'EOR')
+
+    @classmethod
+    def get_word_length(cls, command: str, parameters: str) -> int:
+        """
+        >>> Eor.get_word_length('EOR.B', 'D0, D1')
+        1
+
+        >>> Eor.get_word_length('EOR.W', 'D3, (A4)+')
+        1
+
+        >>> Eor.get_word_length('EOR.L', 'D1, ($1234).W')
+        2
+
+        >>> Eor.get_word_length('EOR.L', 'D0, ($FFFF).L')
+        3
+
+        >>> Eor.get_word_length('EOR.B', 'D0, -(A3)')
+        1
+
+        >>> Eor.get_word_length('EOR.W', 'D7, (A0)')
+        1
+
+        Gets what the end length of this command will be in memory
+        :param command: The text of the command itself (e.g. "LEA", "MOVE.B", etc.)
+        :param parameters: The parameters after the command
+        :return: The length of the bytes in memory in words, as well as a list of warnings or errors encountered
+        """
+        # Split the parameters into EA modes
+        params = parameters.split(',')
+
+        # src = parse_assembly_parameter(params[0].strip())  # Parse the source and make sure it parsed right
+        dest = parse_assembly_parameter(params[1].strip())
+
+        length = 1  # Always 1 word not counting additions to end
+
+        if dest.mode == EAMode.AWA:  # Appends a word
+            length += 1
+
+        if dest.mode == EAMode.ALA:  # Appends a long, so 2 words
```

Comment:
very petty, but shouldn't this be `elif`?

Reply:
It's not totally necessary since the destination can't be two modes at once, but it might make it more readable if it was `elif`.